base/Base/Base_Base_implementation.F90

```diff
@@ -287,21 +302,27 @@ module subroutine MAPL_FieldAllocCommit(field, dims, location, typekind, &
              allocate(VAR_3D(lb1:ub1, lb2:ub2, UNGRID(1)), STAT=STATUS)
              _VERIFY(STATUS)
              VAR_3D = INIT_VALUE
-             call ESMF_FieldEmptyComplete(FIELD, farrayPtr=VAR_3D, &
+             call ESMF_FieldEmptyComplete(FIELD, farray=VAR_3D, &
+                  indexflag=ESMF_INDEX_DELOCAL, &
                   datacopyFlag = ESMF_DATACOPY_REFERENCE,         &
                   gridToFieldMap=gridToFieldMap,              &
                   totalLWidth=haloWidth(1:griddedDims),     &
                   totalUWidth=haloWidth(1:griddedDims),     &
+                  ungriddedLBound=[1],&
+                  ungriddedUBound=[ungrid(1)], &
                   rc = status)
```

Comment:
Looks like there are a bunch of these.  Please do a sweep of the file and fix.  (Feel free to make this status stuff a separate PR - just let me know.)

Reply:
I have cleaned the file up to replace all RC/_VERIFY with _RC and all STAT/_VERIFY with _STAT. I will mark them as resolved here and make a new PR just for that.